docs/intro/tutorial.rst

```diff
@@ -379,78 +391,67 @@ Let's add this code to our spider::
 
-Note how we've changed to use the method ``.extract_first()``, which extracts
-the first element from a selector list returned by ``.xpath()``.
+If you run this spider, it will output the extracted data with the log::
 
-Now try crawling quotes.toscrape.com again and you'll see sites being printed
-in your output. Run::
+    2016-09-19 18:57:19 [scrapy] DEBUG: Scraped from <200 http://quotes.toscrape.com/page/1/>
+    {'tags': ['life', 'love'], 'author': 'André Gide', 'text': '“It is better to be hated for what you are than to be loved for what you are not.”'}
+    2016-09-19 18:57:19 [scrapy] DEBUG: Scraped from <200 http://quotes.toscrape.com/page/1/>
+    {'tags': ['edison', 'failure', 'inspirational', 'paraphrased'], 'author': 'Thomas A. Edison', 'text': "“I have not failed. I've just found 10,000 ways that won't work.”"}
 
-    scrapy crawl quotes
 
-Using our item
---------------
+.. _storing-data:
 
-:class:`~scrapy.item.Item` objects are custom Python dicts; you can access the
-values of their fields (attributes of the class we defined earlier) using the
-standard dict syntax like::
-    
-    >>> from tutorial.items import QuoteItem
-    >>> item = QuoteItem()
-    >>> item['text'] = 'Some random quote'
-    >>> item['text']
-    'Some random quote'
+Storing the scraped data
+========================
 
-So, in order to return the data we've scraped so far, the final code for our
-Spider would be like this::
+The simplest way to store the scraped data is by using :ref:`Feed exports
+<topics-feed-exports>`, with the following command::
 
-    import scrapy
-    from tutorial.items import QuoteItem
+    scrapy crawl quotes -o items.json
 
+That will generate an ``items.json`` file containing all scraped items,
+serialized in `JSON`_.
 
-    class QuotesSpider(scrapy.Spider):
-        name = "quotes"
-        start_urls = [
-            'http://quotes.toscrape.com/page/1/',
-            'http://quotes.toscrape.com/page/2/',
-        ]
+You could've also used other formats, like `JSON Lines`_::
 
-        def parse(self, response):
-            for quote in response.xpath('//div[@class="quote"]'):
-                item = QuoteItem()
-                item['text'] = quote.xpath('span[@class="text"]/text()').extract_first()
-                item['author'] = quote.xpath('span/small/text()').extract_first()
-                yield item
+    scrapy crawl quotes -o items.jl
 
+The `JSON Lines`_ format is useful because it's stream-like, you can easily
+append new records to it. As each record is a separate line, you can also
+process big files without having to fit everything in memory, there are tools
+like `JQ`_ to help doing that at the command-line.
 
-Now crawling quotes.toscrape.com yields ``QuoteItem`` objects::
+In small projects (like the one in this tutorial), that should be enough.
+However, if you want to perform more complex things with the scraped items, you
+can write an :ref:`Item Pipeline <topics-item-pipeline>`. As with Items, a
```

Comment:
"As with Items" looks odd here, as it has not been introduced yet, only seen in `items.py` in the project file tree